spring-native/src/main/java/org/springframework/aot/test/AotCacheAwareContextLoaderDelegate.java

```diff
@@ -38,22 +42,69 @@
 
 	private final AotContextLoader aotContextLoader;
 
+	public AotCacheAwareContextLoaderDelegate() {
+		this(new AotContextLoader());
+	}
+
 	AotCacheAwareContextLoaderDelegate(AotContextLoader aotContextLoader) {
```

Comment:
This constructor is no longer called by anything as far as I can see, besides the default constructor. It should be removed.

Reply:
Good catch! 👍 

I simply overlooked that.

It's gone now.